packages/vscode-extension/package.nls.json

```diff
@@ -157,6 +157,30 @@
     "teamstoolkit.commandsTreeViewProvider.manageCollaboratorTitle": "Manage Collaborator",
     "teamstoolkit.commandsTreeViewProvider.manageCollaboratorDescription": "Manage M365 Teams App (with Microsoft Entra app) Collaborators",
     "teamstoolkit.commandsTreeViewProvider.addWebpartTitle": "Add SPFx Web Part",
+    "teamstoolkit.commandsTreeViewProvider.officeDevDeployTitle": "Deploy",
+    "teamstoolkit.commandsTreeViewProvider.officeDevDeployDescription": "Deploy",
+    "teamstoolkit.commandsTreeViewProvider.publishAppSourceTitle": "Publish",
+    "teamstoolkit.commandsTreeViewProvider.publishAppSourceDescription": "Link to the wiki about how to publish the add-in to AppSource",
+    "teamstoolkit.commandsTreeViewProvider.createOfficeAddInTitle": "Create a New Add-in",
```

Comment:
Form here we can only create a new Office Add-in not other types of apps? That is vert strange, what if I had an Add-in opened but I want to create a Teams app?

Reply:
We'll update this to leverage the existing "Create a New App" string.

Reply:
@MuyangAmigo  @hermanwenhe  I've changed it to "Create a New App". Thanks for your suggestion.

Reply:
@MuyangAmigo Sorry that the original PR is closed by mistake and it couldn't be reopened currently. Would you please help review the new one? https://github.com/OfficeDev/TeamsFx/pull/11004